.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.linting.mypyEnabled": false
+}
```

Comment:
Please don't commit developer environment stuff. Or at the very least, put it in a separate PR for it's own consideration.